common/djangoapps/util/milestones_helpers.py

```diff
@@ -175,6 +175,19 @@ def fulfill_course_milestone(course_key, user):
         milestones_api.add_user_milestone({'id': user.id}, milestone)
 
 
+def remove_course_milestone(course_key, user):
+    """
+    Marks the course specified by the given course_key as in-complete for the given user.
```

Comment:
I think we should rename this method to "remove_course_milestones" (plural).  In addition, the docstring should be more general -- here there is too much context about the downstream effects for one particular use case -- and we should emphasize that this operation removes ALL milestones for a given course+user combination.